From 6bdcebb725ed82da90cee4c56a13556d4280dcf0 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Thu, 18 Feb 2016 10:18:15 +0200
Subject: [PATCH 13/46] ARM: dts: imx7d: cl-som-imx7: add 'depends' property to
 gpmi-nand

Add a new 'depends' property to the gpmi-nand node.
The purpose is to create a dependency between devices.
The ecspi1 bus driver with its subnodes has to be configured first
in order to provide the correct mtd device order registration.

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm/boot/dts/imx7d-cl-som-imx7.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d-cl-som-imx7.dts b/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
index 1c62842..97ba463 100644
--- a/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
+++ b/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
@@ -190,6 +190,7 @@
 &gpmi {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_gpmi_nand>;
+	depends = "/soc/aips-bus@30800000/spba-bus@30800000/ecspi@30820000";
 	status = "disabled";
 	nand-on-flash-bbt;
 
-- 
1.9.1

